schema/methods_directory/regression/kernel_ridge/data_per_property.json

```diff
@@ -0,0 +1,40 @@
+{
+    "schemaId": "methods-directory-regression-linear-data-per-property",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "linear regression parameters schema",
+    "properties": {
+        "xFit": {
+            "description": "training data",
+            "type": "array"
+        },
+        "dualCoefficients": {
+            "description": "dual coefficients",
+            "type": "array"
+        },
+        "perFeature": {
+            "type": "array",
+            "description": "per-feature (property used for training the ML method/model) parameters",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "name": {
+                        "description": "feature name in 'flattened' format",
+                        "type": "string"
+                    },
+                    "importance": {
+                        "description": "pvalue: https://en.wikipedia.org/wiki/P-value",
+                        "type": "number"
+                    }
```

Comment:
Name and importance seem to be more general and are not only relevant for the kernel ridge - should be included elsewhere